README.md

```diff
@@ -99,7 +99,7 @@ python -m pip install -U pipeline-ai
 This project is made with poetry, [so firstly setup poetry on your machine](https://python-poetry.org/docs/#installation).
 
 Once that is done, please run
-
+    chmod +x setup.sh
```

Comment:
We can do this to the actual file in the repository and commit it, then users won't have to do it themselves.

Reply:
did not know git would track this